backend/installation/README.md

```diff
@@ -29,6 +37,12 @@ $ git clone git@github.com:YOUR-GITHUB-USERNAME/Nitro-Backend.git
 {% endtab %}
 {% endtabs %}
 
```

Comment:
@Tirokk I would love to see our git remote added as `upstream` here:
```
git remote add upstream git@github.com:Human-Connection/Nitro-Backend.git
```
or HTTPS:
```
git remote add upstream https://github.com/Human-Connection/Nitro-Backend.git
```

Reply:
That’s a good idea @roschaefer . Hope I done it right with a comment ahead.